Anytype/Sources/PresentationLayer/Modules/Membership/Models/MembershipStatusExtension.swift

```diff
@@ -0,0 +1,40 @@
+import Services
+import Foundation
+
+
+public extension MembershipStatus {
+    static var empty: MembershipStatus {
+        return MembershipStatus(
+            tier: nil,
+            status: .statusUnknown,
+            dateEnds: .distantFuture,
+            paymentMethod: .methodCard
+        )
+    }
+    
+    var formattedDateEnds: String {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "d MMM y"
```

Comment:
Please support localization. Your method broken format for different languages. For example: Chinese and USA.

Reply:
For example:
```
    private let dateFormatter: DateFormatter = {
        let dateFormatter = DateFormatter()
        dateFormatter.dateStyle = .medium
        dateFormatter.timeStyle = .none
        return dateFormatter
    }()
```